doc/nrf/libraries/bluetooth_services/services/ddfs.rst

```diff
@@ -0,0 +1,61 @@
+.. _ddfs_readme:
+
+Direction and Distance Finding Service (DDFS)
+#############################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Bluetooth® LE GATT Direction and Distance Finding Service is a custom service that provides publication of distance, azimuth and elavation measurement data.
+Additional support for measurement configuration.
+
+Service UUID
+************
+
+The 128-bit vendor-specific service UUID is 21490000-494a-4573-98af-f126af76f490.
+
+Characteristics
+***************
+
+This service has the following characteristics.
+
+Distance Measurement Characteristic (21490001-494a-4573-98af-f126af76f490)
```

Comment:
Are you sure you need to have the UUID in every title?

Reply:
I did this similar to the NUS service documentation. Maybe uuid should not be in the title.
It seems to me that writing a uuid is helpful to the user. Because it is a custom service.

Reply:
Maybe we could get rid of that string of numbers in both docs. It looks awful and should be clear from the beginning of the document.

Reply:
In the Service UUID section I have posted a table that contains the UUID of the characteristics . What do you think about it?

Reply:
Dunno really whether it is useful as the UUID is the same for all the listed characteristics anyway. I guess we can live without the table as the UUID is stated in the beginning already. You could maybe say that all the characteristics use the same one.

Reply:
The UUIDs are not the same, but are very similar. This is a custom service, so I think posting the UUID will make it easier to use.